mono/mini/interp/interp.c

```diff
@@ -2888,6 +2888,72 @@ static int opcode_counts[512];
 		} \
 	} while (0);
 
+static
+#ifndef DISABLE_REMOTING
```

Comment:
Why guarding here ? Let's never inline this method.

Reply:
It only has the address-taken local under this ifndef.
Otherwise it inlines reasonably and contributes nothing to the caller's stack usage.
That is the reason. If it is too confusing, or if you are concerned about the inlined code bytes polluting cache, I can see that. But for stack usage, when there is no remoting,..